pylot/control/mpc/utils.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
```

Comment:
Since this is being used just inside the `compute_curvature` function, we should move the import there. Or just use `math` instead of `np` :)

Reply:
Done